frontend/src/component/common/PremiumFeature/PremiumFeature.tsx

```diff
@@ -70,22 +73,22 @@ export interface PremiumFeatureProps {
 }
 
 export const PremiumFeature = ({ feature, tooltip }: PremiumFeatureProps) => {
-    const { url, plan } = PremiumFeatures[feature];
+    const { url, plan, label } = PremiumFeatures[feature];
 
     const tracker = usePlausibleTracker();
 
     const handleClick = () => {
         tracker.trackEvent('upgrade_plan_clicked', {
-            props: { feature },
+            props: { label },
```

Comment:
Depends on whether we want to show this as `feature` or `label` in Plausible, but not showing it as `feature` seems clearer to me.
```suggestion
            props: { feature: label },
```